pkg/kubernetes/scripts/kube-client-cockpit.js

```diff
@@ -804,8 +804,9 @@
 
     .factory("cockpitKubectlConfig", [
         '$q',
+        "$timeout",
         'cockpitRunCommand',
-        function($q, runCommand) {
+        function($q, $timeout, runCommand) {
```

Comment:
This isn't being used anywhere, did you mean to use it in the `kubectl version` call?

Reply:
Leftover cruft from my first approach. I'll remove.